internal/idp/keycloak/const.go

```
@@ -18,6 +18,10 @@ package keycloak
 // realm and is the only client we interact with for role assignments.
 const realmManagementClientID = "realm-management"
 
+// authorizationClientID is the clientId of the Keycloak client that has Authorization Services enabled.
+// This client manages authorization resources for Projects and other protected resources.
+const authorizationClientID = "osac-authorization"
```

golangci-lint (2.12.2)

[error] const authorizationClientID is unused

(unused)